2015/music-and-measure-theory/german/sentence_translations.json

```diff
@@ -668,7 +668,7 @@
  {
   "input": "However, think of the savant who finds harmonic patterns in all rational numbers.",
   "model": "nmt",
-  "translatedText": "Denken Sie jedoch an den Gelehrten, der in allen rationalen Zahlen harmonische Muster findet.",
+  "translatedText": "Aber erinnere dich an den Experten, der in allen rationalen Zahlen harmonische Muster findet.",
```

Comment:
Alternativ to `Experte` could be `Begabter`

Reply:
In line 284 the translation went with `Experte` (gm), so I kinda just rolled with it.
Wouldn't be too much of a hassle to change it to `Begabter`, it's only 4 occurences. 

But I find it a bit odd using `Begabter` while the script talks about an acquired sense for patterns instead of an inherent one `Line 255: "patterns your ear has been trained to pick up on."`

Reply:
Good point. `Experte` is a better choice.